BangazonAPI/Models/Training.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Training
```

Comment:
The names of the models should match the database tables-- this should be TrainingProgram